src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --main-color: #00ACDC;
+}
+
+.nav-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.header .container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+}
+
+.nav-list li {
+  display: inline-block;
+  padding: 0;
+  margin: 0;
+}
+
+.nav-logo {
   margin: 0;
+  padding: 0;
+  font-size: 0;
+}
+
+.nav-link {
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin-right: 20px;
```

Comment:
it's better to move it to parent element where you put this one. There are margins between them
between nav-items i mean.